docs/components/concepts/backups.md

```diff
@@ -12,29 +12,47 @@ A Camunda 8 SaaS backup consists of a data backup of Zeebe, Operate, Tasklist, O
 
 With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes, and should not be used for archival of process data.
 
-## Backup and cluster relationship
-
+:::caution
 Backups are created and managed on a per-cluster basis. It is important to be aware that deleting a cluster will also delete all associated backups.
 
-:::caution
 Exercise caution when deleting clusters to avoid unintended loss of backups.
 :::
 
-Your cluster generation needs to be greater or equal to `8.2.4` to support backups.
+## Backup types
+
+### Manual backup
+
+Manual backups refer to the user-initiated process of creating a consistent snapshot of the state of all system components, including Zeebe, Operate, Tasklist, and Optimize. These backups are managed on a per-cluster basis and are primarily designed for disaster recovery purposes.
+
+Your cluster generation needs to be greater or equal to `8.2.4` to support manual backups.
 
-## Rate limits
+#### Backup limits and rate limits
 
-Each cluster has a limit of three backups. To ensure system stability, backup operations are subject to rate limits. Specifically, you can perform a backup operation every five hours. However, it is possible to delete a backup and create a new one before the 5-hour rate limit expires.
+Each cluster has a limit of three manual backups. To ensure system stability, backup operations are subject to rate limits. Specifically, you can perform a backup operation every five hours.
```

Comment:
Can I contact support if I want to increase this limit?

Reply:
Or CSM like scheduled backups?

Reply:
If I am not mistaken, this is a limit on the Console side, we can ask @ultraschuppi or @ColRad if this is possible

Reply:
Thats currently a hardcoded limit.